### ISSUE TITLE
As user I want to be able to log in with username and password

### ISSUE THREAD

**Issue by nidalaa** [lonelyplanet/backpack-ui#370]
On the login page I can click `Or sign in with a legacy username and password`.
However, I can only login using my `email + password`, when I try `login + password` I am being redirected to https://www.lonelyplanet.com/?error=Invalid%20username%2C%20email%20or%20password.

@toddkazakov  was able to find a root reason in backpack https://lonelyplanet.slack.com/archives/C5GS26QE6/p1499444912304948

**Comment by nidalaa** [comment]
@nickfranciosi there are rumors that you are already on it;)

**Comment by nickfranciosi** [comment]
@nidalaa yep! should be fixed with https://github.com/lonelyplanet/backpack-ui/pull/371, we need to up bump the backpack version and then update the package json in all the apps that use it. Should be ready soon. I'll let you know! Thanks

**Comment by nidalaa** [comment]
Thank **you** 🙇
